### DISCUSSION TITLE
API update interval seems very low by default?

### DISCUSSION THREAD

**Discussion by thuesen** [albaintor/homeassistant_electrolux_status#22] [General]
Thanks for forking the deleted repo. I just installed this and I hope it can work to give us a heads up about when the washing cycle is done :)

I noticed that the "API update interval (seconds)" seems to be set to 30 as default. 

May I suggest to increase this value, to try and avoid annoying Electrolux with too many requests?

Personally I've set it to 600 (10 minutes). Ideally I wouldn't mind if it was dynamic. For my use case, 1 hour would be fine, and once it detects an active wash cycle, then lower it to maybe 5 or 10 minutes.  When the machine is not running, I don't need frequent updates about what it's not doing :D

**Comment by albaintor** [comment]
Hi,

the API update interval is not used anymore as I use the websocket interface : websocket is a persistent 2 way communications, so that any changes are pushed by the server. It may not be instantly because the appliance first have to push the changes to the server and then the server back to us through the websocket.
So you shouldn't get any significant delays to recover updates, unless the server has problems of course

**Reply by thuesen** [reply]
Okay thanks for the explanation. It appeared when I click on “config” for the integration so that’s why I figured it was used.
